static/app/utils/issueTypeConfig/errorConfig.tsx

```diff
@@ -111,6 +116,21 @@ const errorHelpTypeResourceMap: Record<
       linksByPlatform: {},
     },
   },
+  [ErrorHelpType.MODULE_NOT_FOUND]: {
+    resources: {
+      description: tct(
+        '[errorTypes] occur in Next.js applications when an imported module cannot be accessed. To learn more about how to fix these errors, check out these resources:',
```

Comment:
Is the wording `Module not found: Can't resolve` specific to nextjs? If not (if this is a webpack or generic JS error), I'm curious why we're restricting this answer to applying to only nextjs apps.

Reply:
he sentry answer we are linking to is more directed at NextJS (https://sentry.io/answers/module-not-found-nextjs/) instead of being a more generic JS answer